.github/workflows/ci-actions.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         uses: joschi/setup-jdk@b9cc6eabf7e7e3889766b5cee486f874c9e1bd2d
         with:
           java-version: 11
+          release: jdk8u242-b08
```

Comment:
Do we need this?

Reply:
... in a step that installs Java __11__? :-)